doc/users/next_whats_new/margin_getters.rst

```diff
@@ -0,0 +1,4 @@
+Getters for xmargin, ymargin and zmargin
+------------------------------------------------------------------
```

Comment:
Ah, sorry. I haven't written these before so I didn't know! I'll commit this version, then?

Reply:
By eye, that line still looks too long to me.  Basically the dashes need to be the same length as the title.

Reply:
I think I should've done the previous commit by myself instead of via the github UI because it created a merge conflict. Not sure if what I did to solve it is right, but I think it's resolved in the latest commit, with the dashes being at the same level.